integr8ly/integreatly-operator
# MGDAPI-3799 Fix gosec G306

# Issue link
https://issues.redhat.com/browse/MGDAPI-3799

# What
Lowered permissions for file in test function and removed gosec command from the make command

# Verification steps
- Run `make gosec/exclude` and make sure there are no issues.
- Prow tests are passing in the PR so no other verification is needed.
